code/game/objects/items/weapons/melee/energy.dm

```diff
@@ -294,3 +294,73 @@
 			host.embedded -= src
 			host.drop_from_inventory(src)
 		spawn(1) if(src) qdel(src)
+
+/*
+ *Energy Spear
+ */
+
+ /obj/item/weapon/melee/energy/spear

```

Comment:
Is there a space before the / here? B/c if so that's what Travis is likely shouting about

Reply:
..
Thank you.